packages/engine/src/avatar/functions/moveAvatar.test.tsx

```diff
@@ -128,16 +127,16 @@ describe('moveAvatar function tests', () => {
         parentUUID: getComponent(sceneEntity, UUIDComponent),
         position: new Vector3(),
         rotation: new Quaternion(),
-        entityUUID: Engine.instance.userID as string as EntityUUID,
+        entityUUID: Engine.instance.store.userID as string as EntityUUID,
         avatarURL: '',
         name: ''
       })
     )
 
     applyIncomingActions()
 
-    spawnAvatarReceptor(Engine.instance.userID as string as EntityUUID)
-    const entity = AvatarComponent.getUserAvatarEntity(Engine.instance.userID)
+    const entity = UUIDComponent.getEntityByUUID(Engine.instance.store.userID as string as EntityUUID)
+    spawnAvatarReceptor(Engine.instance.store.userID as string as EntityUUID)
```

Comment:
i wonder if instead of running spawnAvatarReceptor here we should ensure the event sourcing state reactors work properly (which call it at runtime)

Reply:
The useEffect that runs this post event sourcing is locked behind an isClient check. Going to try removing it and see what happens. Otherwise will need to stick with manually calling spawnAvatarReceptor

Reply:
Aha, yeah should be fine to remove that check.